### PR TITLE
support PSENet MindIR inference

### PR DESCRIPTION
**Pull request by VictorHe-1**
Thank you for your contribution to the MindOCR repo.
Before submitting this PR, please make sure:

- [x] You have read the [Contributing Guidelines on pull requests](https://github.com/mindspore-lab/mindcv/blob/main/CONTRIBUTING.md)
- [x] Your code builds clean without any errors or warnings
- [x] You are using approved terminology
- [ ] You have added unit tests

## Motivation

support PSENet MindIR inference and add inference tutorial in readme, fix some typo in documents.

## Test Plan

(How should this PR be tested? Do you require special setup to run the test or repro the fixed bug?)

## Related Issues and PRs

(Is this PR part of a group of changes? Link the other relevant PRs and Issues here. Use https://help.github.com/en/articles/closing-issues-using-keywords for help on GitHub syntax)

### CODE REVIEW

File: deploy/py_infer/src/data_process/postprocess/det_postprocess.py [hunk 1]
```diff
@@ -4,10 +4,35 @@
...
+            binary_thresh=binary_thresh,
+            box_thresh=box_thresh,
+            min_area=min_area,
+            box_type=box_type,
+            scale=scale,
+            rescale_fields=rescale_fields)
+
+    def __call__(self, pred, shape_list):
+        pred = pred[0]  # only need the first output
+        return super().__call__(pred, shape_list)
```
Comment (liangxhao) [inline deploy/py_infer/src/data_process/postprocess/det_postprocess.py:38]: 基类DetBasePostprocess的__call__定义了这个pred可以支持tuple，所以这个应当挪到PSEPostprocess基类中处理，否则很多网络的后处理都要专门重载一次这个操作
if isinstance(pred, tuple):
    pred = pred[0]

File: configs/det/psenet/README.md [hunk 2]
```diff
@@ -150,6 +150,46 @@ To evaluate the accuracy of the trained model, you can use `eval.py`. Please set
 python tools/eval.py --config configs/det/psenet/pse_r152_icdar15.yaml
 ```
 
+### 3.6 Inference
```
Comment (HaoyangLee) [inline configs/det/psenet/README.md:153]:
```suggestion
### 3.6 MindSpore Lite Inference
```

File: deploy/py_infer/src/data_process/postprocess/det_postprocess.py (hunk 1 above)
Comment (VictorHe-1) [reply deploy/py_infer/src/data_process/postprocess/det_postprocess.py:38]: 好的

File: configs/det/psenet/README.md (hunk 2 above)
Comment (VictorHe-1) [reply configs/det/psenet/README.md:153]: ok

File: deploy/py_infer/src/data_process/postprocess/det_postprocess.py (hunk 1 above)
Comment (VictorHe-1) [reply deploy/py_infer/src/data_process/postprocess/det_postprocess.py]: 已修改

**Review by HaoyangLee — APPROVED** [review]

**Review by SamitHuang — APPROVED** [review]

**Pull request merged**